tcp_queue_length/README.md

```diff
@@ -58,7 +56,7 @@ Then, the check can be activated by setting the `datadog.systemProbe.enableTCPQu
 
 ### Metrics
 
-For each TCP connection, `tcp_queue_length` collects the minimum and the maximum amount of data in the send and receive buffers between each data collection time point.
+For each container or a pod, tcp_queue_length integration returns the connection read/write buffer which is the mostly used. For example, if a container has three TCP connections, for which their read buffers are used 40%, 25% and 80% respectively, the metric `tcp_queue.read_buffer_max_usage_pct` will return 0.8
```

Comment:
```suggestion
For each container of a pod, tcp_queue_length integration returns usage in percent of the connection read/write buffer which is the mostly used. For example, if a container has three TCP connections, for which their read buffers are used 40%, 25% and 80% respectively, the metric `tcp_queue.read_buffer_max_usage_pct` will return 0.8
```

Reply:
Will remove "pod" because the integration also applies to plain docker and ECS containers.